linkerd/cache/src/cache.rs

```diff
@@ -109,7 +105,7 @@ where
     ///
     /// Polls the underlying `DelayQueue`. When elements are returned from the
     /// queue, remove the associated key from `values`.
-    pub fn purge(&mut self) {
+    pub fn poll_purge(&mut self) {
```

Comment:
nit/tioli: it still seems to me like a method named `poll_foo` should return a `Poll`...